### ISSUE TITLE
Supported? 

### ISSUE THREAD

**Issue by leotsarev** [SergeyTeplyakov/ExceptionAnalyzer#23]
Hi Sergey,
I'm started making something like that https://github.com/leotsarev/hardcode-analyzer/tree/master/Tsarev.Analyzer.Exceptions
and after that I stumbled upon your project.
Will you willing to accept some pull requests and package your analyzer for Nuget.org?
Appveyor is free-of-charge solution to CI for open source C#.

**Comment by SergeyTeplyakov** [comment]
I definitely will accept PRs.

**Comment by jzabroski** [comment]
This makes me so happy!

**Comment by leotsarev** [comment]
@SergeyTeplyakov what do you plan on ErrorProne.Net? Does ErrorProne.Net implements superset of this analyzer or not? Do you think that both projects will be useful on long run?

**Comment by leotsarev** [comment]
Please see #24 #25 & #26 in this order

**Comment by SergeyTeplyakov** [comment]
> @SergeyTeplyakov what do you plan on ErrorProne.Net? Does ErrorProne.Net implements superset of this analyzer or not? Do you think that both projects will be useful on long run?

Yes, ErrorProne.NET is a superset. I want to create a set of analyzers there: 

* Core analyezrs
* Exception analyzers
* Memory analyzers
* Structs analyzers (my main focus right now).

**Comment by leotsarev** [comment]
So, basically it's no reason for me to look into ExceptionAnalyzer instead of ErrorProne.Net?
Do you have a Nuget build of ErrorProne?

**Comment by SergeyTeplyakov** [comment]
> Do you have a Nuget build of ErrorProne?

Not yet:(
